toqito/state_props/is_mutually_unbiased_basis.py

```diff
@@ -80,17 +78,25 @@ def is_mutually_unbiased_basis(vec_list: list[np.ndarray | list[float | Any]]) -
     :return: :code:`True` if :code:`vec_list` constitutes a mutually unbiased basis, and
              :code:`False` otherwise.
     """
-    if len(vec_list) <= 1:
+    if len(vectors) <= 1:
         raise ValueError("There must be at least two bases provided as input.")
```

Comment:
Should we make an issue for checking what happens when `is_check_something` has been provided an empty array as an input? 

For the current function, I think the output will be `true` if 2 or more empty arrays are provided as input. In the tests, I do see 1 function checking for the `ValueError` on line 82.

Reply:
Hmm, actually, I'm not sure this check is necessary. A vector of length 1 isn't necessarily an error but is just not a MUB. I've removed this check and adapted the tests accordingly. Thanks for catching that!